internal/jimm/permissions/permissionmanager.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2024 Canonical.
+
+// The permissions package provides business logic for handling user permissions.
+package permissions
+
+import (
+	"github.com/juju/names/v5"
+
+	"github.com/canonical/jimm/v3/internal/db"
+	"github.com/canonical/jimm/v3/internal/errors"
+	"github.com/canonical/jimm/v3/internal/openfga"
+)
+
+// permissionManager provides a means to manage roles within JIMM.
+type permissionManager struct {
+	store   *db.Database
+	authSvc *openfga.OFGAClient
+	// JIMM's UUID and tag
+	uuid string
+	tag  names.ControllerTag
+}
+
+// NewPermissionManager returns a new permission manager that provides
+// permission handling and resolution of JAAS tags.
+func NewPermissionManager(store *db.Database, authSvc *openfga.OFGAClient, uuid string, tag names.ControllerTag) (*permissionManager, error) {
```

Comment:
suggestion: rename to NewManager.. that way whoever calls this will do `permissions.NewManager` instead of `permissions.NewPermissionManager` (the word permission is repeated)